src/olympia/api/jwt_auth.py

```diff
@@ -63,14 +66,25 @@ def jwt_decode_handler(token, get_api_key=APIKey.get_jwt_key):
         'require_iat': True,
         'require_nbf': False,
     }
+
     try:
+        now = timegm(datetime.utcnow().utctimetuple())
+
         payload = jwt.decode(
             token,
             api_key.secret,
             options=options,
             leeway=api_settings.JWT_LEEWAY,
             algorithms=[api_settings.JWT_ALGORITHM]
         )
+
+        # Verify clock skew for future iat-values pyjwt removed that check in
+        # https://github.com/jpadilla/pyjwt/pull/252/
+        # `verify_iat` is still in options because pyjwt still validates
+        # that `iat` is a proper number.
+        if int(payload['iat']) > (now + api_settings.JWT_LEEWAY):
```

Comment:
oh, I see that you moved it over explicitly. I think that's a good thing to do, at least until we know can assess the usefulness of `iat` validation with security folks.

r+